openedx/features/classrooms/admin.py

```diff
@@ -1,9 +1,15 @@
+"""
+Admin file for classrooms application
+"""
 from django.contrib import admin
 
-from models import DiscussionCommunityMembership
+from .models import DiscussionCommunityMembership
 
 
 class DiscussionCommunityMembershipAdmin(admin.ModelAdmin):
+    """
+    A Admin class for discussion community membership
```

Comment:
Just "Admin class for discussion community membership" would be fine. Please remove "A "

Reply:
Done